.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,16 +15,16 @@ Please provide the following information:
 
 **- What I did**
```

Comment:
🤔 Looks like our templates still refer to a 404; https://docs.docker.com/opensource/code/

This one also links to https://github.com/docker/cli/blob/master/CONTRIBUTING.md - which very likely was a copy/paste from me at the time (as we had some internal tool complaining there was no template in this repo)